app/views/pages/moving-guide/conus.html.erb

```diff
@@ -1,9 +1,9 @@
-<%- @page_title = 'What to Expect with a CONUS Move' -%>
+<%- @page_title = 'What to Expect' -%>
```

Comment:
Do we really want to remove CONUS here? Maybe the navigation should be changed instead?